CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# Contributing to the P4 Compiler Project (p4c)
+
+Thank you for considering contributing to the P4 Compiler Project (p4c)! Your contributions are valuable and help improve the project for everyone. Before getting started, please take a moment to review the following guidelines.
+
+## Coding Standard Philosophy
+
+Please note that this project adheres to the [P4 Coding Standard Philosophy](https://github.com/p4lang/p4c/blob/main/docs/CodingStandardPhilosophy.md). By participating, you are expected to uphold this code. 
+
+## How to Contribute
+We welcome and appreciate new contributions. Check out [git usage](https://github.com/p4lang/p4c/tree/main/docs#git-usage) to get started.
+
+### Guidelines 
+
+* Writing unit test code [Guidelines](https://github.com/p4lang/p4c/tree/main/docs#adding-new-test-data).
+* Write documentation [Guidelines](https://github.com/p4lang/p4c/tree/main/docs#writing-documentation).
+* [Coding conventions](https://github.com/p4lang/p4c/tree/main/docs#coding-conventions).
+* Opening pull requests and writing commit messages [Guidelines](https://github.com/p4lang/p4c/blob/main/docs/CodingStandardPhilosophy.md#Git-commits-and-pull-requests).
+* Code has to be reviewed before it is merged.
+* Make sure all tests pass when you send a pull request.
+* Participate in the code review process and address any feedback or comments.
+* Make sure `make cpplint` produces no errors (`make check` will also run this)
```

Comment:
This is outdated. It should say: 
`cmake --build . --target clang-format cpplint black isort`

Reply:
Thank you for the information.